imprint-de.html

```diff
@@ -0,0 +1,228 @@
+					<p>Anmeldedaten: Um sich für die Mailingliste anzumelden, reicht es aus, wenn Sie Ihre E-Mailadresse angeben. Optional bitten wir Sie, Ihren Realnamen oder einen Nickname anzugeben.<p>
+					<p>Kündigung/Widerruf - Sie können den Empfang unserer Mailingliste jederzeit kündigen, d.h. Ihre Einwilligungen widerrufen. Einen Link zur Kündigung der Mailingliste finden Sie hier: <a href="https://lists.openrailwaymap.org/lists/listinfo/openrailwaymap">https://lists.openrailwaymap.org/lists/listinfo/openrailwaymap</a></p>
+
+					<p><strong>Onlinepräsenzen in sozialen Medien</strong></p>
+
+					<p>Wir unterhalten Onlinepräsenzen innerhalb sozialer Netzwerke und Plattformen, um mit den dort aktiven Interessenten und Nutzern kommunizieren und sie dort über unsere Leistungen informieren zu können. Beim Aufruf der jeweiligen Netzwerke und Plattformen gelten die Geschäftsbedingungen und die Datenverarbeitungsrichtlinien deren jeweiligen Betreiber.</p>
+					<p>Soweit nicht anders im Rahmen unserer Datenschutzerklärung angegeben, verarbeiten wir die Daten der Nutzer sofern diese mit uns innerhalb der sozialen Netzwerke und Plattformen kommunizieren, z.B. Beiträge auf unseren Onlinepräsenzen verfassen oder uns Nachrichten zusenden.</p>
+
+					<p><strong>Einbindung von Diensten und Inhalten Dritter</strong></p>
+
+					<p>Wir setzen innerhalb unseres Onlineangebotes auf Grundlage unserer berechtigten Interessen (d.h. Interesse an der Analyse, Optimierung und wirtschaftlichem Betrieb unseres Onlineangebotes im Sinne des Art. 6 Abs. 1 lit. f. DSGVO) Inhalts- oder Serviceangebote von Drittanbietern ein, um deren Inhalte und Services, wie z.B. Karten einzubinden (nachfolgend einheitlich bezeichnet als “Inhalte”).<p>
+					<p>Dies setzt immer voraus, dass die Drittanbieter dieser Inhalte, die IP-Adresse der Nutzer wahrnehmen, da sie ohne die IP-Adresse die Inhalte nicht an deren Browser senden könnten. Die IP-Adresse ist damit für die Darstellung dieser Inhalte erforderlich. Wir bemühen uns nur solche Inhalte zu verwenden, deren jeweilige Anbieter die IP-Adresse lediglich zur Auslieferung der Inhalte verwenden.</p>
+
+					<p><strong>OpenStreetMap</strong></p>
+
+					<p>Wir binden die Landkarten des Dienstes <a href="https://www.openstreetmap.org/">OpenStreetMap</a> ein, die durch die OpenStreetMap Foundation, 132 Maney Hill Road, Sutton Coldfield, West Midlands B72 1JU, United Kingdom (OSMF) angeboten werden. Beim Abruf der Karten werden Kartenkacheln von Servern der OSMF geladen, die sich im Vereinigten Königreich und anderen Ländern befinden.</p>
+					<p>Wir haben keinen Einfluss auf die Verarbeitung der Daten durch die OSMF. Weitere Informationen finden sich in der <a href="https://wiki.osmfoundation.org/wiki/Privacy_Policy">Datenschutzerklärung der OSMF</a>.</p>
```

Comment:
Paypal content is not loaded any more because this pull request replaces the button by a simple link. In the next steps of the redesign there will be a separate "Donations" page with detailled information about Paypal and FOSSGIS donations.

I was not sure if the fact that Paypal processes data of users who are sending money to us needs any information in the Privacy Policy.

Reply:
Ok, if the third party content is removed, we don't have to mention it in this section. However, we should mention that donations via Paypal will use Paypal. The same applies to FOSSGIS e.V. It is sufficient to mention them and link to their privacy policies.